gae/js/app/views/sitecaptain.js

```diff
@@ -68,11 +68,15 @@ define(
                 m.destroy();
                 this.render();
             },
-            addCaptain: function() {
-                console.log('add captain ', this);
+            addCaptain: function(e) {
+                e.preventDefault();
                 var self = this;
+                var choices = this.form.fields.models.find(function(model){return model.attributes.name == 'captain';});
+                var captain_id = this.model.attributes.captain;
+                this.captain  = _.findWhere(choices.changed.options, {value: captain_id});
```

Comment:
this is choices.attributes.get('options')

this.captain can be a variable used in the closure below ("var captain = " .... )